client/src/pages/download/index.js

```diff
@@ -1,28 +1,44 @@
-import React, { useEffect, useState } from 'react'
+import React, { useEffect, useRef, useState } from 'react'
 import { Box, Text } from 'grommet'
-import { useResponsive } from 'hooks/useResponsive'
+import { useScrollPosition } from 'hooks/useScrollPosition'
 import { useDatasetManager } from 'hooks/useDatasetManager'
+import { useResponsive } from 'hooks/useResponsive'
 import { DatasetSummary } from 'components/DatasetSummary'
 import { DatasetDownloadFileSummary } from 'components/DatasetDownloadFileSummary'
 import { DatasetProjectCard } from 'components/DatasetProjectCard'
 import { Loader } from 'components/Loader'
 import Error from 'pages/_error'
 
 const Download = () => {
+  const { restoreScrollPosition } = useScrollPosition()
   const { myDataset, errors, getDataset, isDatasetDataEmpty } =
     useDatasetManager()
   const { responsive } = useResponsive()
 
   const [loading, setLoading] = useState(true)
 
+  const isMyDatasetFetched = useRef(false) // Prevent re-fetching
```

Comment:
Do we really want to prevent re-fetching? What is the side effect if we were to allow it to fetch?